src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Configuration/GeneralSettings.cs

```diff
@@ -171,5 +171,10 @@ public string GetOrganisationRepositoryLocation
         /// Take care not to upload "old" certificates.
         /// </remarks>
         public int JwtSigningCertificateRolloverDelayHours { get; set; }
+
+        /// <summary>
+        /// Gets or sets the initial organisation harvest date time.
+        /// </summary>
+        public DateTime InitialHarvestDateTime { get; set; } = new DateTime(0);
```

Comment:
Never use the constructor of value types. 
```suggestion
        public DateTime InitialHarvestDateTime { get; set; } = DateTime.MinValue;
```